Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerController : MonoBehaviour {
```

Comment:
I think this class is doing too much. It handles input, destroying itself, hp, ....
It should really be separated into components.